modules/trainingclient-api-job-related-functions.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: REFERENCE
+
+[id="trainingclient-api-job-related-functions_{context}"]
+= TrainingClient API: Job-related functions
+
+[role='_abstract']
+Use these functions to find job-related information.
+
+.List all training job resources
+
+[source,subs="+quotes"]
+----
+client.list_jobs(namespace=__<namespace>__)
+----
+
+.Get information about a specified training job
+
+[source,subs="+quotes"]
+----
+client.get_job("pytorch-distributed-trainer",__<namespace>__,"PyTorchJob")
```

Comment:
"pytorch-distributed-trainer" is a PyTorchJob name, so thinking whether to replace it by `<PyTorchJob name>` placeholder.